addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties

```diff
@@ -80,7 +86,10 @@ channel-type.netatmo.dateMaxTemp.label = Zeitpunkt Max. Temperatur
 channel-type.netatmo.dateMaxTemp.description = Zeigt den Zeitpunkt an, an dem die maximale Temperatur gemessen wurde.
 
 channel-type.netatmo.temperatureTrend.label = Temperatur Trend
-channel-type.netatmo.temperatureTrend.description = Zeigt den Temperatur Trend (z.B. "up", "down" oder "stable") an.
+channel-type.netatmo.temperatureTrend.description = Zeigt den Temperatur Trend (z.B. "Steigend", "Steigend" oder "Fallend") an.
```

Comment:
This should be "Temperaturtrend" in one word.
"Steigend" is there twice.
"Stabil" is missing. ;)